ProjectProtocol/project-protocol-nextjs
# Fix not-found page

Something about having a root layout without <html> tags was breaking things before the not-found page could render, so unrecognized url routes were throwing errors rather than gracefully showing the not-found page.

[Sentry issue](https://project-protocol-hh.sentry.io/issues/6040645760/?environment=production&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=0)

### Testing
1. Visit https://projectprotocol.org/es-MX/abcd1234 and see a blank page (broken)
2. Visit https://deploy-preview-68--project-protocol-staging.netlify.app/es-MX/abcd1234 and see the not-found page rendered as expected
